runtime/Cargo.toml

```diff
@@ -310,16 +310,18 @@ wip = [
     "beefy-light-client",
     "beefy-light-client-runtime-api",
     "evm-fungible-app",
-    "jetton-app",
-    "bridge-channel",
     "pallet-beefy",
     "pallet-beefy-mmr",
     "xor-fee/wip",
-    "soratopia",
     "vested-rewards/wip",
 ]
 
-stage = ["framenode-chain-spec/stage"]
+stage = [
```

Comment:
looks like only `framenode-chain-spec` has the feature

Reply:
Yes, because other pallets is optional and it's enough to just enable them